Makefile

```diff
@@ -4,12 +4,14 @@ export DEVELOPMENT?=false
 export AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
 export AWS_REGION?=us-west-2
 export IMAGE_REPO?=$(AWS_ACCOUNT_ID).dkr.ecr.$(AWS_REGION).amazonaws.com
+export KOPS_CLUSTER_NAME=$(shell cat /dev/urandom | LC_ALL=C tr -dc '[:alpha:]' | tr '[:upper:]' '[:lower:]' | head -c 32).prod.test.pdx.kops-ci.model-rocket.aws.dev
+export TEST_ROLE_ARN=arn:aws:iam::$(AWS_ACCOUNT_ID):role/$(TEST_ROLE_NAME)
 BASE_IMAGE_TAG?=$(shell cat EKS_DISTRO_BASE_TAG_FILE)
 export BASE_IMAGE?=$(IMAGE_REPO)/eks-distro/base:$(BASE_IMAGE_TAG)
 KUBE_BASE_TAG?=v0.4.2-ea45689a0da457711b15fa1245338cd0b636ad4b
 export KUBE_PROXY_BASE_IMAGE?=$(IMAGE_REPO)/kubernetes/kube-proxy-base:$(KUBE_BASE_TAG)
 export GO_RUNNER_IMAGE?=$(IMAGE_REPO)/kubernetes/go-runner:$(KUBE_BASE_TAG)
-ARTIFACT_BUCKET?=my-s3-bucket
+ARTIFACT_BUCKET?=artifactsstack-3794122512-artifactsbucket2aac5544-1f3dgu9wrpiz2
```

Comment:
I would rather this value be in the prowjob and not on the Makefile that the customer runs as well.